rusqlite_migration/src/lib.rs

```diff
@@ -635,19 +653,18 @@ impl<'m> Migrations<'m> {
     ///
     /// Returns [`Error::MigrationDefinition`] if no migration is defined.
     pub fn to_latest(&self, conn: &mut Connection) -> Result<()> {
-        let v_max = self.max_schema_version();
+        let v_max = self.latest_schema_version();
         match v_max {
-            SchemaVersion::NoneSet => {
+            0 => {
                 warn!("no migration defined");
                 Err(Error::MigrationDefinition(
                     MigrationDefinitionError::NoMigrationsDefined,
                 ))
             }
-            SchemaVersion::Inside(v) => {
```

Comment:
I would prefer to keep the `SchemaVersion` type here. It's not just any integer, it's a schema version and those are off by one, effectively, from the index in the migrations vector.